docs/articles/developer-info/NUnit-Copyright-Notice.md

```diff
@@ -1,5 +1,7 @@
 # NUnit Copyright Notice
 
+## NUnit
```

Comment:
Suggest changing main heading to NUnit Copyright Notices and making secondary headings refer to specific projects like NUnit Framework, NUnit 3 Visual Studio Adapter, etc. However the list doesn't have to be complete for this PR to be merged... we can add more as needed.